content/en/docs/reference/node/kubelet-checkpoint-api.md

```diff
@@ -0,0 +1,54 @@
+---
+content_type: "api_reference"
+title: Kubetlet Checkpoint API
+weight: 10
+---
+
+## Kubelet Checkpoint API
+
+Checkpointing a container is the functionality to create a stateful copy of a
+running container. The stateful copy of the container can be moved to another
+system and after restore the container will continue to run at exactly the same
+point it was checkpointed. The concept of checkpointing a container and
+restoring a container can be seen as pausing and unpausing a container with the
+possibility to transfer the container to another system between pausing and
+unpausing.
+
+## Operations {#Operations}
+
+### `post` checkpoint the specified container
+
+Tell the kubelet to checkpoint the specified container.
+
+The kubelet will request a checkpoint from the underlying
+{{<glossary_tooltip term_id="cri" text="CRI">}} implementation. In the checkpoint
+request the kubelet will specify the name of the checkpoint archive as
+`<containerID>.tar` and also request to store the checkpoint archive in the
+`checkpoints` directory below its root directory (as defined by `--root-dir`).
+This defaults to `/var/lib/kubelet/checkpoints`.
+
+The content of the checkpoint archive depends on the underlying CRI implementation.
+
+#### HTTP Request
+
+POST /checkpoint/{namespace}/{pod}/{container}
+
+#### Parameters
+
+- **namespace** (*in path*): string, required
+
+  {{< glossary_tooltip term_id="namespace" >}}
+
+- **pod** (*in path*): string, required
+
+  {{< glossary_tooltip term_id="pod" >}}
+
+- **container** (*in path*): string, required
+
+  {{< glossary_tooltip term_id="container" >}}
+
+#### Response
+
+200: OK
+
+401: Unauthorized
```

Comment:
What happens if
- the feature isn't enabled?
- the container runtime doesn't support checkpointing?
- the checkpoint archive already exists?
- there is no storage left?

For alpha, it might be OK to say that the kubelet can also return other statuses, and maybe put in an HTML comment to say that this should be filled out properly for beta.

Reply:
Also added more details.